ads/_config.js

```diff
@@ -534,6 +534,13 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'insticator': {
+    prefetch: 'https://testthisshit.online/amp-embed-lib/insticator.js',
+    preconnect: [
+      'https://d3lcz8vpax4lo2.cloudfront.net'
+    ]
+   },
```

Comment:
would you please also implement renderStart API?

in your remote loaded `insticator.js`, it should call `context.renderStart` once content is ready to show. This basically gives AMP runtime signal when to hide loading indicator and show your content.

Reply:
renderStart implemented

Reply:
👍verified amp runtime is getting this signal.